src/main/java/com/jcabi/github/User.java

```diff
@@ -244,6 +264,29 @@ public UserEmails emails() {
             return this.user.emails();
         }
 
+        /**
+         * Returns this user's notifications.
+         * @todo Implement the call "List your notifications" as described at
```

Comment:
correct the format, I think all the todos from this class could be merged into one and placed in class description